packages/edit-site/src/components/sidebar-navigation-screen-navigation-item/index.js

```diff
@@ -51,6 +51,11 @@ export default function SidebarNavigationScreenNavigationItem() {
 					icon={ pencil }
 				/>
 			}
+			description={
+				postType === 'page'
+					? __( 'This is a static page.' )
+					: __( 'This is your posts page' )
+			}
 			content={
 				<>
 					{ post?.link ? (
```

Comment:
Not a big deal, but I think it might be better if the URL appears above the description, closer to the title.

Reply:
I actually prefer it below, in fact I consider it a little bit of an awkward affordance to have just for some of the types and not others. I might even suggest removing this link — though that's not at all a strong opinion. Rather, I think we need a separate "view on frontend" affordance somewhere, that's general. Might even be a external link icon that appears when hovering or focusing the frame, or a permanent button, something like that.